app/parse.py

```diff
@@ -9,7 +13,33 @@ class Quote:
 
 
 def main(output_csv_path: str) -> None:
-    pass
+    i = 1
+    all_the_quotes = []
+    while True:
+
+        site = requests.get(f"https://quotes.toscrape.com/page/{i}").content
+        soup = BeautifulSoup(site, "html.parser")
+        all_quotes = soup.select('.quote')
+        for quote in all_quotes:
+            all_tags = []
+            text = quote.select_one('.text').text
+            author = quote.select_one('.author').text
+            tags = quote.select('.tag')
+            for tag in tags:
+                tag_content = tag.text
+                all_tags.append(tag_content)
+            quote = Quote(text=text, author=author, tags=all_tags)
+            one_quote = [quote.text, quote.author, quote.tags]
+            all_the_quotes.append(one_quote)
+        i += 1
+        next = soup.find('nav').find("li", {"class": "next"})
+        if not next:
+            break
+    student_header = ['name', 'age', 'major', 'minor']
+    with open(output_csv_path, 'w', encoding='utf-8') as file:
+        writer = csv.writer(file)
+        writer.writerow(student_header)
+        writer.writerows(all_the_quotes)
```

Comment:
Split this big function into a few smaller ones, stick to the Single responsibility principle.